recipe/meta.yaml

```diff
@@ -35,7 +34,7 @@ requirements:
     - jinja2
     - jupyter_client >=5.3.4
     - jupyter_core >=4.6.1
-    - nbclassic >={{ min_nbclassic }}
+    - nbclassic >=0.4.7
     - nbconvert >=5
     - nbformat
     - nest-asyncio >=1.5
```

Comment:
upstream has depenedecies on:
"jupyter_server>=2.0.1,<3",
    "importlib-resources>=5.0;python_version<\"3.9\"",
    "jupyterlab>=4.0.0a37,<5",
    "jupyterlab_server>=2.19.0,<3",
    "notebook_shim>=0.2,<0.3",
    "tornado>=6.2.0"
see https://github.com/jupyter/notebook/blob/main/pyproject.toml ,or I miss something here ?

Reply:
@M-Waszkiewicz-Anaconda you are looking at the main branch which is already probably ahead of the v6.5.3 release. Here is the toml file for the v6.5.3 we are updating to:
https://github.com/jupyter/notebook/blob/v6.5.3/pyproject.toml

And runtime dependencies are specified in the setup.py file:
https://github.com/jupyter/notebook/blob/v6.5.3/setup.py#L107

Reply:
ahh, OK, that explains